doc/man/8/cephadm.rst

```diff
@@ -194,6 +194,16 @@ Arguments:
 * [--skip-firewalld]           Do not configure firewalld
 * [--skip-pull]                do not pull the latest image before adopting
 
+Configuration:
+
+When starting the shell, cephadm looks for configuration in the following order:
+
+1. If the user has provided a custom configuration file (``-c`` option) use it
+2. Use daemon configuration if a daemon ``--name`` parameter has been provided
+3. If ``/var/lib/ceph/<fsid>/config`` directory exists use it
+4. Otherwise find the ``mon`` daemon conf file and use it
+5. Finally: fallback to the default file ``/etc/ceph/ceph.conf``
```

Comment:
```suggestion
1. An explicit, user provided path to a config file (``-c/--config`` option)
2. Config file for daemon specified with ``--name`` parameter (/var/lib/ceph<fsid>/<daemon-name>/config)
3. ``/var/lib/ceph/<fsid>/config/ceph.conf`` if it exists
4. The config file for a ``mon`` daemon (/var/lib/ceph/<fsid>/mon.<mon-id>/config) if it exists
5. Finally: fallback to the default file ``/etc/ceph/ceph.conf``
```
NIT, personal opinion on the wording of this. Just my opinion on it, change whatever you agree with.